packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -290,3 +292,5 @@ module.exports = {
     hints: false,
   },
 };
+
+module.exports = applyPlugins(base, ['typescript'], { path, paths });
```

Comment:
This is controlled by prettier, but I agree that it's a good idea.
We're going to be adding more meta information (and probably move where this is) -- we will need plugin name and plugin version supported: `{ name: 'typescript', version: '1.x' }`.